test/integration/lib/testhelpers/helpers.go

```diff
@@ -93,7 +93,7 @@ func SetupCluster() error {
 // Clean up
 func Cleanup(testName string) {
 	localenvCmd(fmt.Sprintf("dumpSymphonyLogsForTest '%s'", testName), "")
-	localenvCmd("destroy all", "")
+	localenvCmd("destroy all,nowait", "")
```

Comment:
I understand we cannot use wait because we skip the clean up of workload. But I think we still need to wait for symphony pod to be gone? Needn't we?